tools/explorer/lib/tests/model_test.py

```diff
@@ -0,0 +1,114 @@
+"""Testing module for model.py"""
+from absl.testing import absltest
+
+from lib.model import StandardField
+from lib.model import EntityTypeField
+from lib.model import Match
+from lib.model import StandardizeField
+from lib.tests import test_constants
+
+from yamlformat.validator.entity_type_lib import EntityType
+from yamlformat.validator.external_file_lib import RecursiveDirWalk
+from yamlformat.validator import presubmit_validate_types_lib
+from yamlformat.validator import namespace_validator as nv
+
+class StandardFieldTest(absltest.TestCase):
+  def setUp(self):
+    self.test_standard_field = StandardField(
+        namespace_name='',
+        standard_field_name='supply_air_flowrate_sensor'
+    )
+
+  def testGetNamespaceName(self):
+    expected_output = ''
+    function_output = self.test_standard_field.GetNamespaceName()
+
+    self.assertEqual(function_output, expected_output)
+
+  def testGetStandardFieldName(self):
+    expected_output = 'supply_air_flowrate_sensor'
+    function_output = self.test_standard_field.GetStandardFieldName()
+
+    self.assertEqual(function_output, expected_output)
+
+class EntityTypeFieldTest(absltest.TestCase):
+  def setUp(self):
+    self.test_entity_type_field = EntityTypeField(
+        namespace_name='',
+        standard_field_name='supply_air_flowrate_sensor',
+        is_optional=False,
+        increment=''
```

Comment:
what happens when there is an increment here?

Reply:
Good idea.